compiler-rt/include/xray/xray_interface.h

```diff
@@ -97,27 +97,50 @@ enum XRayPatchingStatus {
 /// for possible result values.
 extern XRayPatchingStatus __xray_patch();
 
+extern XRayPatchingStatus __xray_patch_object(int32_t ObjId);
```

Comment:
Non-blocking: Is it possible to provide documentation to this and other added functions?